### PR TITLE
[CLA] Corporate signature for VK DATA ApS

### PR DESCRIPTION
**Pull request by kenneth-vkd**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/cla/corporate/vk-data-aps.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Signed,
+
+Visti Kristensen visti@vkdata.dk https://github.com/visti-vkd
+
+List of contributors:
+
+Kenneth Hansen kenneth@vkdata.dk https://github.com/kenneth-vkd
+Simon Andersen simon@vkdata.dk https://github.com/simon-vkd
+Mads Søndergaard mads@vkdata.dk https://github.com/mads-vkd
+Jacob Sørensen @vkdata.dk https://github.com/jacob-vkd
```
Comment (mart-e) [inline doc/cla/corporate/vk-data-aps.md:18]: Part of the email is missing ?

Comment (kenneth-vkd) [reply doc/cla/corporate/vk-data-aps.md]: Done

**Comment by mart-e** [comment]
Thanks for the update but could you squash the commits to have only one to merge ? :)

**Comment by kenneth-vkd** [comment]
Should be done now, but not sure if it is done correctly

**Comment by mart-e** [comment]
@kenneth-vkd if you check the PR, it still lists 2 commits in the "Commits" tab. You can use [rebase interactive](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history) to do so and then push force. Make sure to keep only one commit message.
Thanks

**Comment by kenneth-vkd** [comment]
@mart-e it seems as I we have broken something completely on our end as I am unable to squash the commits.
The fastest resolution would be to delete this PR and then I will make a new one, where the data is correct from the start

**Pull request closed without merging**